packages/theme-addresses/loader.ts

```diff
@@ -0,0 +1,97 @@
+var query = "query countries($locale: SupportedLocale!) {"
+  + "  countries(locale: $locale) {"
+  + "    name"
+  + "    code"
+  + "    labels {"
+  + "      address1"
+  + "      address2"
+  + "      city"
+  + "      company"
+  + "      country"
+  + "      firstName"
+  + "      lastName"
+  + "      phone"
+  + "      postalCode"
+  + "      zone"
+  + "    }"
+  + "    formatting {"
+  + "      edit"
+  + "    }"
+  + "    zones {"
+  + "      name"
+  + "      code"
+  + "    }"
+  + "  }"
+  + "}";
+
+var GRAPHQL_ENDPOINT = 'https://country-service.shopifycloud.com/graphql';
+
+export interface Country {
+  name: string;
+  code: string;
+  formatting: {
+    edit: string;
+  }
+  labels: {
+    address1: string;
+    address2: string;
+    city: string;
+    company: string;
+    country: string;
+    firstName: string;
+    lastName: string;
+    phone: string;
+    postalCode: string;
+    zone: string;
+  }
+  zones: {
+    name: string;
+    code: string;
+  }[]
+}
+export function loadCountries(locale: string): Promise<Country[]> {
+  var response = fetch(GRAPHQL_ENDPOINT, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*',
+    },
+    body: JSON.stringify({
+      query: query,
+      operationName: 'countries',
+      variables: {
+        locale: toSupportedLocale(locale),
+      },
+    }),
+  });
+
+  return response
+    .then(function(res) { return res.json() })
+    .then(function(countries) { return countries.data.countries });
+}
+
+var DEFAULT_LOCALE = 'EN';
+export var SUPPORTED_LOCALES = [
```

Comment:
That part will go away pretty soon. We'll handle that on the backend. cc @froyomuffin

Reply:
T-T

Reply:
Handled